generators/client/templates/angular/webpack/webpack.dev.js.ejs

```diff
@@ -77,14 +77,6 @@ module.exports = webpackMerge(commonConfig({ env: ENV }), {
             loaders: 'tslint-loader',
             exclude: ['node_modules', new RegExp('reflect-metadata\\' + path.sep + 'Reflect\\.ts')]
         },
-        {
-            test: /\.ts$/,
-            loaders: [
-                'angular2-template-loader',
```

Comment:
is this dependency also removed from package.json ?

Reply:
Nope because we need it in the webpack.test :(